src/views/List.vue

```diff
@@ -1,40 +1,54 @@
 <template>
   <div>
+    <br>
```

Comment:
use CSS instead of line breaks. Line breaks should **only** be used inside text blocks.